backend/app/api/v1/endpoints/ocp/graph.py

```diff
@@ -243,10 +243,13 @@ def jobFilter(pdata: dict, data: dict):
         return []
     columns = ['uuid','jobConfig.jobIterations']
     pdf = pd.json_normalize(pdata)
-    pick_df = pd.DataFrame(pdf, columns=columns)
+    # for jsons without a jobConfig.jobIteration value, json_normalize() 
+    # fills in Not a Number (NaN)
+    pick_df = pd.DataFrame(pdf, columns=columns).dropna(subset=['jobConfig.jobIterations'])
```

Comment:
This seems like overkill and insufficient protection at the same time?

We seem to only care about finding "a" value of `iterations`, and we're arbitrarily grabbing the first row ... which suggests we think they're all the same. You're dropping all the null values here, and then assuming that there's still at least one non-null value to pick. What if there isn't? Would it perhaps be better to use something like a `max` filter, which would return a single value we'd hope would not be null? (And what if *all* the values were missing?)

Reply:
I believe I have addressed this. In order to avoid arbitrarily picking one record I used all of them to match with [pandas.isin()](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.isin.html#pandas.DataFrame.isin).